packages/plugin-rax-app/src/base.js

```diff
@@ -11,8 +11,7 @@ const { isWebpack4 } = require('@builder/compat-webpack4');
 module.exports = (api, { target, babelConfigOptions, progressOptions = {}, isNode }) => {
   const { context, onGetWebpackConfig } = api;
   const { rootDir, command, userConfig } = context;
-  const { experiments = {} } = userConfig;
-  const { exportsField } = experiments;
+  const mpa = userConfig[target] && (userConfig[target].subPackages || userConfig[target].mpa);
```

Comment:
这里的逻辑稍微有点长？这种需要先判断对象中某个 key 是否存在的情况用 `??` 可读性更好些
```suggestion
  const mpa = userConfig[target] ?? (userConfig[target].subPackages || userConfig[target].mpa);
```